Automated testing in WordPress/Codeception.md

```diff
@@ -1,4 +1,4 @@
-Codeception collects and shares the best practices and solutions for testing PHP web applications. With a flexible set of included modules, tests are easy to write, easy to use, and easy to maintain. Codeception encourages developers and QA engineers to concentrate on testing and not on building test suite.
+Codeception collects and shares the recommended practices and solutions for testing PHP web applications. With a flexible set of included modules, tests are easy to write, easy to use, and easy to maintain. Codeception encourages developers and QA engineers to concentrate on testing and not on building test suite.
```

Comment:
Codeception collects and shares recommended practices and solutions for testing PHP web applications. With a flexible set of included modules, tests are easy to write, easy to use, and easy to maintain. Codeception encourages developers and QA engineers to concentrate on testing and not on building test suite.